### DISCUSSION TITLE
Importing webradio channel lists from radio-browser.info

### DISCUSSION THREAD

**Discussion by Raitsa** [rern/rAudio#637] [Feature Requests]
Hey guys.

Having started with the awesome rAudio-1 on my Rpi 4 very recently, I sorta miss a handy way of importing channels from the listener maintained https://www.radio-browser.info

I downloaded a .pls file of some metal rock stations but no luck importing them to my rAudio in .pls or m3u.

Some details:

https://i.imgur.com/rNT4F1W.png

https://i.imgur.com/s9gn2TC.png

I wish this kind of import would be possible. Or is it ;-)

**Comment by Raitsa** [comment]
I downloaded .pls and .m3u from this page:

https://www.radio-browser.info/search?page=1&order=clickcount&reverse=true&hidebroken=true&tagList=metal

**Comment by cmh714** [comment]
I never use the pls file and always pull the real url out of it in an editor and then use that.....not really an answer for you but a workaround for now...GL! That website has the actual Url in the details I see....

**Comment by AndyFx12** [comment]
If you're on Android, try "Radiodroid"-App. It is based on radiobrowser.info and can play every listed station with rAudio's MPD. 
Once you've seen them on rAudio's playlist, you can save them easily with one click to your rAudio webradios.

Andy

**Comment by Raitsa** [comment]
That Radiodroid trick is cool, I had no idea such a way exists, thanks @AndyFx12 
It helps somewhat but still hoping there would be an easier way built in the r-Audio1.

**Comment by hjfisch** [comment]
Thanks @AndyFx12 .
Working great .

**Comment by Raitsa** [comment]
Hifiberry64 alpha 8 now offers radiobrowser support.

![radiobrowser](https://github.com/rern/rAudio/assets/8920478/0de99dc4-a6c9-4139-a553-af22f191e104)

**Comment by Raitsa** [comment]
Here what the radio-browser search now looks like in Hifiberry64 alpha 8.

Search for almost 50 000 web radio stations by a search term yle:
![1](https://github.com/rern/rAudio/assets/8920478/aec10bc9-684b-4c72-be26-e049d1dc3212)

![2](https://github.com/rern/rAudio/assets/8920478/0b9db781-88dd-4f72-b2eb-40b823bcfe43)

Search with the term flac:
![8](https://github.com/rern/rAudio/assets/8920478/5c0eabf4-f10f-4d24-a385-4ec5962b8370)

As you can see, you can find any web radio station easily, as long as you know some letters about its name.
And this makes finding and adding new stations to your favourites very easy.

It is not yet possible in Hifiberry to search with tags, language, country or genre. 
But those search terms do work on the Radio-browser.info webpage search very well.
